### PR TITLE
Update 06-wherka-ama.md

### PR DESCRIPTION
**Pull request by wherka-ama**

### CODE REVIEW

**Review by claire-gallesio — COMMENTED** [review]
Hello Waldek

File: _slides/06-wherka-ama.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Hello there :wave:
+Hello there :wave: from 🇵🇱 and 🏴󠁧󠁢󠁥󠁮󠁧󠁿
```
Comment (claire-gallesio) [inline _slides/06-wherka-ama.md]: Are you sure about the flags ?

**Review by claire-gallesio — APPROVED** [review]

File: _slides/06-wherka-ama.md (hunk 1 above)
Comment (Ramakrishna-Mandepudi) [reply _slides/06-wherka-ama.md]: yes

**Review by Ramakrishna-Mandepudi — APPROVED** [review]

**Review by riezebosch — CHANGES_REQUESTED** [review]
use ascii emoticons instead

**Review by riezebosch — APPROVED** [review]

**Comment by wherka-ama** [comment]
> use ascii emoticons instead

grrrrrr.... happy now?!
:smile:

**Review by imtyezjh — APPROVED** [review]

**Pull request merged**